jupyter-book/trajectories/pseudotemporal.ipynb

```diff
@@ -15,53 +15,95 @@
    ]
```

Comment:
<p>"This procedure is justified by the observation that dynamical processes progress on a low-dimensional manifold."</p><p>This sentence almost warrants a citation :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/multimodal-single-cell-best-practices/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Yes, agreed. Missed that one.</p>